.gitignore

```
@@ -0,0 +1,6 @@
+# .gitignore 
+/node_modules
+/dist
+/.vscode
+/_DS_Store
+/src/images/_DS_Store
```

Можно лучше: игнорировать файлы без привязки к директории.
```suggestion
_DS_Store
.DS_Store
```